src/argilla/client/feedback/utils.py

```diff
@@ -111,10 +111,10 @@ def feedback_dataset_in_argilla(
 
     if (name and workspace) or name:
         if workspace is None:
-            workspace = rg.Workspace.from_name(rg.active_client().get_workspace())
+            workspace = rg.Workspace(rg.active_client().get_workspace())
```

Comment:
That means that `Workspace.__init___` will call internally the argilla API, right? I would prefer to have this operation outside of an object constructor. I would say to separate the `Workspace` class for the data model and `Workspaces` class to interact with workspaces in Argilla.
```python
import argilla as rg 

workspace= Workspaces.from_name("my-workspace")
workspace.id # UUID(......)
workspace.name # my-workspace

new_workspace=Workspaces.create(Workspace(name="new-workspace")
new_workspace.id # UUID (...)
new_workspace.name # new-workspace
```

Reply:
This is what we had before, but it's a little bit weird because users will be able to register workspaces that don't exist in Argilla, and this functionality is just intended to be used within Argilla, but if you think it's better to keep the init simpler then we can just do nothing on init, and keep the `from_name` which is still there